addon/mixins/node-actions.js

```diff
@@ -144,7 +142,14 @@ export default Ember.Mixin.create({
             });
             // After user has been saved, add user as a contributor
             return user.save().then(user => {
-                this.send('addContributor', user.id, permission, isBibliographic);
+                var node = this.get('_node');
```

Comment:
Also, this whole method might change this week if APIv2 changes for adding unregistered contributors go through, allowing contributors to be created with an email/fullname combo instead of the userid. If that is the case, I will put in a follow-up PR.